graql/language/BUILD

```diff
@@ -20,7 +20,6 @@ package(default_visibility = ["//visibility:public"])
 load("@graknlabs_dependencies//tool/checkstyle:rules.bzl", "checkstyle_test")
 
 files = [
-    "compute.feature",
```

Comment:
Let's not delete this file. It's a placeholder for us to do compute queries in the (near) future.